cs/src/core/Index/CheckpointManagement/DeviceLogCommitCheckpointManager.cs

```diff
@@ -180,36 +147,11 @@ public byte[] GetCommitMetadata(long commitNum)
                 body = writePad;
             else
                 ReadInto(device, 0, out body, size + sizeof(int));
-            
-            if (!overwriteLogCommits)
```

Comment:
don't we need to dispose of the device when we are done?

Reply:
"using var device" should handle it